app/views/facilities/index.html.haml

```diff
@@ -13,11 +13,28 @@
     - @recently_used_facilities.each do |facility|
       %h4.header-tight= link_to facility, facility_path(facility)
     %hr
-    %h4= text(".all")
 
+  - if @recent_products.any?
+
+    %h4= text(".recently_purchased")
+    .product_list
+      %ul
+        - @recent_products.each do |product|
+          %li
+            = link_to "#{product.name}" + " - #{product.facility}", facility_product_path(product.facility, product)
```

Comment:
Nitpicky, but I would do this all in one string: `"#{product.name} - #{product.facility}"